run.go

```diff
@@ -151,8 +151,15 @@ func (r *runCmd) run(c *cli.Context) error {
 
 // TODO: share all this stuff with the Docker driver in server or better yet, actually use the Docker driver
 func runff(ff *funcfile, stdin io.Reader, stdout, stderr io.Writer, method string, envVars []string, links []string, format string, runs int) error {
-	sh := []string{"docker", "run", "--rm", "-i", fmt.Sprintf("--memory=%dm", ff.Memory)}
-
+	startFunc := []string{
+		"docker", "run",
+		fmt.Sprintf("--name=%s", ff.Name),
```

Comment:
Won't this limit you to one running instance of a function at a time?

Reply:
it will, posting new patch to use hash to get a unique name here